pkg/innerring/rpc.go

```diff
@@ -136,14 +136,14 @@ func (c *ClientCache) GetHeader(task *audit.Task, node *netmap.Node, id *object.
 	headParams.WithMainFields()
 	headParams.WithAddress(objAddress)
 
-	addr, err := network.HostAddrFromMultiaddr(node.Address())
+	addr, err := network.AddressFromString(node.Address())
 	if err != nil {
 		return nil, fmt.Errorf("can't parse remote address %s: %w", node.Address(), err)
 	}
 
 	cli, err := c.Get(addr)
 	if err != nil {
-		return nil, fmt.Errorf("can't setup remote connection with %s: %w", addr, err)
+		return nil, fmt.Errorf("can't setup remote connection with %s: %w", addr.String(), err)
```

Comment:
I don't think you need to manually call `String()` method when argument is substituted to `%s` modifier.